Bug Fixes and Issue Resolution in Tuning-Halving

alannair opened the pull request:
This PR is analogous to PR [#374](https://github.com/vhive-serverless/vSwarm/pull/374). It incorporates the following into the tuning-halving benchmark.
- Applies Updated Knative Runners. Closes [#267](https://github.com/vhive-serverless/vSwarm/issues/267).
- In a [previous commit](https://github.com/vhive-serverless/vSwarm/commit/e3547166e6e776101569d12fd5fd4e3221a61034), the github workflow manifests had not been updated. This caused the _Test Docker Compose_ job to fail, as the environment variable `BUCKET_NAME` had not been configured, causing an empty string to be passed to the function (see [log](https://github.com/vhive-serverless/vSwarm/actions/runs/3533960680/jobs/5970282187)). This has also been fixed in this PR. 
  Note that `os.getenv('ENV_VAR', 'DEFAULT_VAL')` returns `DEFAULT_VAL` only if `ENV_VAR` is not set. This does not happen if `ENV_VAR` is set to an empty string, which occurs when `ENV_VAR` is created (in `docker-compose.yaml`) but not set to a value.
- Updated Readme, Placed repeated environment variables in the workflow at a single place, etc.

dhschall reviewed (approved):
It looks al right but maybe @shyamjesal knows more

alannair commented:
@dhschall @shyamjesal I updated the PR description (it had become outdated) and reverted the unnecessary changes I made in the Readme in a new commit. Code changes are the same as before.
